cortex-mixin/config.libsonnet

```
@@ -26,7 +26,7 @@
     // These are used by the dashboards and allow for the simultaneous display of
     // microservice and single binary cortex clusters.
     job_names: {
-      ingester: '(ingester|cortex$)',
+      ingester: '(ingester|ingester-blocks|cortex$)',  // Match also ingester-blocks, which is used during the migration from chunks to blocks.
```

Can we use `ingester.*`?

```suggestion
      ingester: '(ingester.*|cortex$)',  // Match also ingester-blocks, which is used during the migration from chunks to blocks.
```

---

We could but we may include other custom jobs/deployments we don't want. Using `ingester-blocks` may be safer.

---

Why wouldn't we want all kind of ingesters?